bad_connector_woocommerce/components/backend_adapter.py

```diff
@@ -237,7 +237,9 @@ def read(self, external_id=None, attributes=None):
 
     def create(self, data):
         """Creates the data in remote"""
-        result = self._call(self._woo_model, data, http_method="post")
+        resource_path = "{}/{}/refunds".format(self._woo_model, data["order_id"])
```

Comment:
Let's override this method into refund-related adapter, it will create a problem otherwise when we supports the different exports functionality in woo-commerce